src/Microsoft.DocAsCode.DataContracts.Common/TocItemViewModel.cs

```diff
@@ -111,12 +111,12 @@ public string NameForVB
 
         [ExtensibleMember]
         [JsonIgnore]
-        public Dictionary<string, object> Additional { get; set; } = new Dictionary<string, object>();
+        public Dictionary<string, object> Metadata { get; set; } = new Dictionary<string, object>();
```

Comment:
OK...hope to break nobody

---
In reply to: [90600251](https://github.com/dotnet/docfx/pull/941#discussion_r90600251) [](ancestors = 90600251)